### DISCUSSION TITLE
Male - Female

### DISCUSSION THREAD

**Discussion by crazyLillo** [tesivo/tesivo_s_VTMB_italian_translation_for_Unofficial_Patch#56] [Domande & Rispsote]
ho letto da qualche parte qui, che parlavate di come avevate risolto il problema dei dialoghi mancanti,
 (il personaggio davanti a noi resta muto), quando si interpreta un personaggio femminile.

Potete darmi più dettagli?

grazie!

**Comment by tesivo** [comment]
Occorre che esistano i corrispondenti file .lip e .mp3 (non ricordo il formato del file audio) del dialogo in questione.
Se il dialogo per il personaggio femminile e maschile sono uguali allora si avrà un'unica coppia (.lip, .mp3) associata alla frase, altrimenti due coppie.

**Reply by crazyLillo** [reply]
Sapevo da tempo quello che hai scritto, ma forse mi sono espresso male.
Ormai, dopo aver installato l'SDK e letto molte cose, ne so abbastanza, ma non tutto.

Intendevo, quando mi accingo a svolgere il tutorial e lo faccio con un personaggio femminile,
jack, dopo 3-4 frasi dette bene, si blocca e non parla quando invece dovrebbe farlo.
Sono andato avanti a vedere se altre frasi si bloccavano ed è stato così.
Ho fatto altre prove, ma niente, jack non parla in certe frasi.
Ho fatto una piccola ricerca e sembra che questo problema è solo nostro (Italiano), e ho detto sembra...

Quindi avendo letto che lo avevate risolto, mi dite esattamente come avete fatto se si gioca con personaggio femminile?

Ho letto che nel file README,md, hai scritto che:
Audio nei dialoghi mancante se si impersona un personaggio femminile: 
si conosce già la soluzione al problema, serve solo del tempo per applicarla. In ogni caso il dialogo funziona comunque.

Qual'è la soluzione insomma?

**Reply by tesivo** [reply]
Ho compreso correttamente il problema che riscontri. La possibile causa più frequente è quella che ti ho descritto qui sopra: file .lip e/o .mp3 mancanti/errati. 

Se non ricordo male però è ancora accaduto che ci fosse questo problema per colpa di qualche spaziatura di troppo all'interno dei file .dlg. Tieni però presente che non è detto che la causa è da ricercarsi nella frase in cui manca il dialogo, a volte l'errore nel file è presente prima e si verifica nel gioco dopo.

**Reply by crazyLillo** [reply]
scusa. ma hai scritto:
si conosce già la soluzione al problema, serve solo del tempo per applicarla.

volevo solo conoscere anchio la soluzione da voi adottata!
te la sto tirando fuori con il contagocce...

Giocando con personaggio maschile, la mia traduzione del tutorial non fa una piega, finisco il tutorial senza un intoppo di qualsiasi genere.
se ci provo con personaggio femminile, jack in alcune frasi rimane muto a guardarmi e va avanti premendo i tasti delle risposte.

ANCORA NON HO CAPITO IL PERCHE'!

**Reply by tesivo** [reply]
Le soluzioni che abbiamo trovato sono le due che ti ho descritto. :D

Se è con la traduzione per ora pubblicata che riscontri il problema mandami un salvataggio precedente al problema e indago di persona. Se invece è all'inizio faccio io ;) (dimmi però le scelte iniziali)

**Reply by crazyLillo** [reply]
No riguarda la mia traduzione e comunque anche prima con la traduzione disponibile, il fatto accadeva lo stesso.

Le tue parole mi hanno messo una pulce nell'orecchio, me la son grattata e questo è il risultato :


Ho risolto alla grande!

In giro nei file dell'SDK avevo trovato questo (lo tradotto in Italiano) :


(((Il file LIP che viene caricato si basa su nome del file audio a cui è associato l'evento speak. Per mantenere la vita 
semplice inseriamo sempre i file VCD, LIP e MP3 nella stessa directory insieme. (e nominarli con la stessa convenzione)))

Poi nell'help dell'ottimo programma per localizzare ho trovato questo (mi era sfuggito, ho tradotto e riporto i pezzi che ci 

interessano inerenti al caso) :

Dice :

(((Si noti che alcuni degli originali file DLG hanno una struttura danneggiata che è necessario riparare manualmente. 

* Il filtro Differenza mostra le righe per l'NPC in cui i campi Maschio e Femmina differiscono. Questo
è importante se vuoi ascoltare il suono di diverse linee femminili nel gioco. Hai bisogno
di annotare i numeri di riga, trovare i rispettivi file nella directory sound e copiare l'intera terzina
di MP3, LIP e VCD dalla variante " col_e to col_f ". Quindi se il filtro Differenza ti dice che
la riga 61 per Vandal è una da copiare, si individuano i file 

`../sound/character/dlg/santa monica/vandal/line_61_col_e. [mp3,lip,vcd] `

e copiali in 

`line_61_col_f.[mp3.lip.vcd] `

nella stessa directory. Devi estrarre tutti i dati dai pacchetti in anticipo però. )))
-------------------------------------------------------------------------------------------------

Quello che bisogna fare per fare le cose corrette è questo :

Iintanto trovare tutti i file mp3 del caso (da sound di Vampire) e ricopiarli nella stessa cartella dell'unofficial patch dove 

risiedono i file lip con gli eventuali file vcd.

poi bisogna copiare da qualche parte tutti i file Lip (..._col_e.lip), e rinominarli con la convenzione "f". Quindi 

(..._col_e.lip) diventa  (..._col_f.lip). ( "f" sta per Female).
una volta fatto bisogna ricopiare tutti questi nuovi file lip (con "f") nella cartella dove già ci sono i file (..._col_e.lip).

la stessa cosa va fatta con i file mp3. tutti i file  (..._col_e.mp3) si ricopiano da qualche parte e si rinominano con la 

convenzione "f" al posto di "e", (..._col_f.mp3).
Quindi riopiare tutti questi nuovi mp3 nella cartella dell'unofficial patch dove risiedono tutti i file lip ( "e" e "f" ), con gli 

eventuali file vcd e tutti i file mp3 ( "e" e "f" ).

In pratica bisogna ricreare tutti i file lip con le risposte al femminile e ricreare tutti i file mp3 per l'evento speak femminile.

Facendo così tutto il tutorial al femminile va alla grande e senza il minimo intoppo.

a breve metterò nella discussione "MY Work" tutti i file aggiornati e altro.
